iOS/Services/StitchLocalMongoDBService/StitchLocalMongoDBService_watchos/StitchLocalMongoDBService_watchos.h

```diff
@@ -0,0 +1,19 @@
+//
+//  StitchLocalMongoDBService_watchos.h
+//  StitchLocalMongoDBService_watchos
+//
+//  Created by Jay Flax on 6/20/18.
```

Comment:
Remove all of these types of headers. We need to do a MongoDB license pass on this repo soon (not now).